compiler/util/ir_pb2.py

```diff
@@ -107,9 +108,12 @@ def set(self, obj, value):
       self._set_value(obj, self._type(**value))
     elif isinstance(value, _Text) and self._decode_names:
       self._set_value(obj, self._type(self._decode_names(value)))
+    elif isinstance(value, _Text) and issubclass(self._type, enum.Enum):
+      self._set_value(obj, getattr(self._type, value))
```

Comment:
Added this path to handle the case where a json serialization had a string representation, eg `BYTE` instead of `8`